tensorflow/python/keras/backend.py

```diff
@@ -1212,24 +1212,29 @@ def zeros(shape, dtype=None, name=None):
   """Instantiates an all-zeros variable and returns it.
 
   Arguments:
-      shape: Tuple of integers, shape of returned Keras variable
-      dtype: String, data type of returned Keras variable
-      name: String, name of returned Keras variable
+      shape: Tuple or list of integers, shape of returned Keras variable
+      dtype: data type of returned Keras variable
+      name: name of returned Keras variable
 
   Returns:
       A variable (including Keras metadata), filled with `0.0`.
       Note that if `shape` was symbolic, we cannot return a variable,
       and will return a dynamically-shaped tensor instead.
 
-  Example:
+  For Example:
```

Comment:
This should be kept as `Example:`, afaik

Reply:
Updated. Thanks for the suggestions!